vercel.json

```
@@ -747,10 +752,9 @@
       "permanent": true
     },
     {
-      "source":"/guides/next-js/internationalization/",
+      "source": "/guides/next-js/internationalization/",
       "destination": "/docs/guides/nextjs-internationalization",
       "permanent": true
```

This redirect isn't working, maybe try removing the trailing '/'?

```suggestion
      "source": "/guides/next-js/internationalization",
      "destination": "/docs/guides/nextjs-internationalization",
      "permanent": truebjz 
```